collect_app/src/main/res/layout/form_hierarchy_layout.xml

```diff
@@ -2,6 +2,7 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/parent_container"
```

Comment:
What's this for?

Reply:
It's not needed anymore I used it when I displayed the snackbar in the fragment but now it is in the activity.